app/views/landing-pages/league/PageLeagueGlobal.vue

```diff
@@ -641,45 +641,101 @@ export default {
+                </div>
+                Cloud Revolver 7.1 Headset
+              </a>
+            </div>
+            <div class="col-sm-4">
+              <a href="https://hyperx.com/collections/gaming-keyboards/products/hyperx-alloy-origins-mechanical-gaming-keyboard" target="_blank" class="prize-section__winners--product-link">
+                <div>
+                  <img src="/images/pages/league/hyperx-keyboard.png" alt="Alloy Origins Keyboard" class="prize-section__winners-1--img">
+                </div>
+                Alloy Origins Keyboard
+              </a>
+            </div>
+            <div class="col-sm-4">
+              <a href="https://hyperx.com/collections/gaming-mice/products/hyperx-pulsefire-fps-pro-gaming-mouse" target="_blank" class="prize-section__winners--product-link">
+                <div>
+                  <img src="/images/pages/league/hyperx-mouse.png" alt="Pulsefire FPS Pro Mouse" class="prize-section__winners-1--img">
+                </div>
+                Pulsefire FPS Pro Mouse
+              </a>
+            </div>
+          </div>
+        </div>
+        <div class="prize-section__winners-2">
+          <div class="col-sm-6">
+            <div class="prize-section__winners-text">2<span class="prize-section__small-top">nd</span> {{ $t('league.position') }}</div>
+            <div class="row">
+              <div class="col-sm-6">
+                <a href="https://hyperx.com/collections/gaming-headsets/products/hyperx-cloud-ii" target="_blank" class="prize-section__winners--product-link">
+                  <div>
+                    <img src="/images/pages/league/hyperx-cloud2-headphones.png" alt="Cloud II Headset" class="prize-section__winners-2--img">
+                  </div>
+                  Cloud II Headset
+                </a>
+              </div>
+              <div class="col-sm-6">
+                <a href="https://hyperx.com/collections/earbuds/products/hyperx-cloud-earbuds" target="_blank" class="prize-section__winners--product-link">
+                  <div>
+                    <img src="/images/pages/league/hyperx-earbuds.png" alt="Cloud Earbuds" class="prize-section__winners-2--img">
+                  </div>
+                  Cloud Earbuds
+                </a>
+              </div>
+            </div>
+          </div>
+<!--          <div class="col-sm-3">&nbsp;</div>-->
+          <div class="col-sm-3 col-sm-offset-3">
+            <div class="prize-section__winners-text">3<span class="prize-section__small-top">rd</span> {{ $t('league.position') }}</div>
+            <a href="https://hyperx.com/collections/gaming-headsets/products/hyperx-cloud-stinger-core-pc" target="_blank" class="prize-section__winners--product-link">
+              <div>
+                <img src="/images/pages/league/hyperx-cloud-stinger-headset.png" alt="Cloud Stinger Core Headset" class="prize-section__winners-2--img">
```

Comment:
This image is not well-cropped and has some non-transparent white at the top right. Here's a quick example of how to optimize images like this before upload in Mac's Preview and [ImageOptim](https://imageoptim.com/mac), showing 1) removing transparent areas with the instant alpha tool, 2) cropping with the rectangular selection tool, and 3) reoptimizing the image for smaller filesize.
![2022-03-26 15 14 01](https://user-images.githubusercontent.com/99704/160258794-db1641d9-db53-4e14-87a9-5362af557153.gif)

Result:
![hyperx-cloud-stinger-headset](https://user-images.githubusercontent.com/99704/160258843-14a81086-5578-41c0-b5d4-44eb8be3ca63.png)

I didn't notice the other images having the first problem, but can you take a whack at doing steps 2-3 each of the uploaded images, to get them properly cropped and their filesize optimized?